src/utils.cpp

```diff
@@ -1516,13 +1516,14 @@ int get_skill(struct char_data *ch, int skill, int &target)
   // Iterate through their bioware, looking for anything important.
   if (should_gain_physical_boosts && ch->bioware) {
     for (struct obj_data *bio = ch->bioware; bio; bio = bio->next_content) {
-      if (GET_BIOWARE_TYPE(bio) == BIO_REFLEXRECORDER && GET_OBJ_VAL(bio, 3) == skill) {
-        strlcat(gskbuf, "Reflex Recorder skill increase: 1. ", sizeof(gskbuf));
-        skill_dice++;
-      } else if (GET_BIOWARE_TYPE(bio) == BIO_ENHANCEDARTIC)
+      if (GET_OBJ_VAL(bio, 0) == BIO_REFLEXRECORDER) {
+        recorder = GET_OBJ_VAL(bio, 5);
```

Comment:
Instead of using a raw value call like this, please create a new reflex recorder block in utils.hpp that has a descriptive name of what this value is. Example from another piece of bioware:

```c++
// Platelet factory extra data.
#define GET_BIOWARE_PLATELETFACTORY_DATA(bioware)       (GET_OBJ_VAL((bioware), 5))
#define GET_BIOWARE_PLATELETFACTORY_DIFFICULTY(bioware) (GET_OBJ_VAL((bioware), 6))
```